src/index.html

```diff
@@ -11,7 +11,134 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="form-group">
+        <legend>Personal information</legend>
+        <label class="form-input">Surname:
```

Comment:
Text should be moved to a new line. Also:

[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) p.2 (`Each level of nesting, including text, contained inside the element, requires 2-space offset.`)
```suggestion
        <label class="form-input">
          Surname:
```